### PR TITLE
Import PageHeader and Logo As Is

### PR DESCRIPTION
**Pull request by sandrahoang686**
## Available PR templates

Related Issue: https://github.com/NASA-IMPACT/veda-ui/issues/1136

- [Default](?expand=1&template=default.md)
- [Version Release](?expand=1&template=version_release.md)
- _Alternatively delete and start empty_

### CODE REVIEW

File: app/components/nav.tsx [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
-interface NavItem {
-  // @TODO: To be replaced with exposed NavItem type from veda-ui
-  path: string;
-  title: string;
-  subItems?: NavItem[];
-}
-
-const navItems: NavItem[] = [
+const navItems = [
+  // @TODO: This should use the NavLinkType from veda-ui...
```
Comment (hanbyul-here) [inline app/components/nav.tsx:7]: is this still to do?

**Comment by hanbyul-here** [comment]
I added a bit of style so the nav can be displayed with full width through this PR: https://github.com/developmentseed/next-veda-ui/pull/9

**Comment by sandrahoang686** [comment]
@hanbyul-here thanks! i didn't bother because I just wanted to export it as is without styling since we will have to go back and revisit styling for template designs. But merged your PR anyways, thank you 🙏🏼

File: app/components/nav.tsx (hunk 1 above)
Comment (sandrahoang686) [reply app/components/nav.tsx:7]: Yes!! this TODO i'm actually working on actively with exposing some of these interfaces from veda-ui. It's meant to stay

**Review by hanbyul-here — APPROVED** [review]

**Review by hanbyul-here — APPROVED** [review]
thanks for working on it~

**Pull request merged**